### ISSUE TITLE
My official semver suggestion

### ISSUE THREAD

**Issue by eligrey** [jacoborus/node-filesaver#8]
If your package manager forces semver (e.g. npm) and you wish to upload my static-API date-versioned repos, use 1.0.YYYYMMDD as the semver[[1]](https://twitter.com/sephr/status/542978804441169920)

**Comment by jacoborus** [comment]
You're right, but this is still a work in progress, and its API is going to change in next version. I just need a few free days to adapt it to my CMS and publish the definitive API.

Are you using this library in a production server?
